File: llm_generated_code/Gemini/CPP/Level1/Q26.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
    int t;
    cin >> t;

    while (t--) {
        int n;
        cin >> n;

        vector<int> sticks(n);
        for (int i = 0; i < n; i++) {
            cin >> sticks[i];
        }

        sort(sticks.begin(), sticks.end());

        int minOps = INT_MAX;
        for (int i = 2; i < n; i++) {
            int target = sticks[i] / 3;
            int ops = 0;
            for (int j = 0; j < 3; j++) {
                ops += abs(sticks[i - j] - target);
            }
            minOps = min(minOps, ops);
        }

        cout << minOps << endl;
    }

    return 0;
}

File: llm_generated_code/Gemini/Python/Level1/Q26.py
def min_operations(lengths):
    n = len(lengths)
    min_ops = float('inf')
    for i in range(n - 2):
        for j in range(i + 1, n - 1):
            target_length = (lengths[i] + lengths[j]) // 2
            ops = 0
            for k in range(n):
                ops += abs(lengths[k] - target_length)
            min_ops = min(min_ops, ops)
    return min_ops

def main():
    t = int(input())
    for _ in range(t):
        n = int(input())
        lengths = list(map(int, input().split()))
        print(min_operations(lengths))

if __name__ == "__main__":
    main()
